Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/elf/ElfConstants.java

```diff
@@ -494,12 +494,12 @@
 	public static final short EM_AMDGPU = 224;
 	/** RISC-V */
 	public static final short EM_RISCV = 243;
-	/** Lanai 32-bit processor */
+	/**{@code  Lanai 32-bit processor}*/
 	public static final short EM_LANAI = 244;
 	/** Linux kernel bpf virtual machine */
 	public static final short EM_BPF = 247;
 
-	/** used by NetBSD/avr32 - AVR 32-bit */
+	/**{@code  used by NetBSD/avr32 - AVR 32-bit}*/
 	public static final short EM_AVR32_unofficial = 0x18ad;
```

Comment:
The numbers were displaying funny in the font. Using the code sets them to fixed width and solved that.